discovery-provider/src/queries/get_usdc_purchases.py

```diff
@@ -20,6 +20,8 @@ class GetUSDCPurchasesCountArgs(TypedDict):
     buyer_user_id: Optional[int]
     content_ids: Optional[List[int]]
     content_type: Optional[PurchaseType]
+    sort_method: Optional[PurchaseSortMethod]
```

Comment:
Remove these, we shouldn't allow sorting for getting the count?

Reply:
This doesn't seem resolved?

Reply:
now it is :)